### DISCUSSION TITLE
DRO Layout request and 3d view before Program view

### DISCUSSION THREAD

**Discussion by runryder912** [terjeio/ioSender#258] [Ideas]
Would it be possible to have a dro layout like this? This was pulled from cncjs.  The X0Y0 Z0 options in there move the machine to it zero position once press. I like the pull down menu for the distance moves which has many options.
![Dro layout idea](https://user-images.githubusercontent.com/81871634/205453660-04f4fd7c-faa3-4454-9aed-a51017f37720.JPG)

What do you think about the 3d view first and then the program view tabs? 

Thanks again for working so hard on this. I made the jump to iosender and soon i saw of the features i needed

**Comment by terjeio** [comment]
Sorry for responding late...

> Would it be possible to have a dro layout like this?

Yes - and perhaps no not as detailed due to screen real estate restrictions. ioSender supports up to 9 axes now (grblHAL 8) so new functionality may have to be made dynamically available. I'll look into this and other suggestions when I put focus on the sender again.
